scipy-1.0/paper.tex

```diff
@@ -1184,28 +1177,30 @@ \section*{Supporting Information}
 \setcounter{figure}{0}
 \setcounter{table}{0}
 
-\begin{figure}[H]
-\centering
-\includegraphics[width=0.8\textwidth]{supporting_info/knn_complexity_confirm}
-\caption{Assessment of \texttt{cKDTree} nearest neighbor query time complexity as a function of the number of input points (N) along with similar assessment for related algorithms in scikit-learn. The calculations are based on code previously reported\cite{knn-jake}, but updated to use SciPy 1.0.0, Python 3.6, NumPy 1.13.1, and Cython 0.27.3.}
-\label{fig:knn-complexity}
-\end{figure}
 
-\begin{figure}[H]
```

Comment:
Can't include screenshots, see #143.

Reply:
Pretty sure you can keep screenshots in SI. The standards for SI are generally far less strict that for the main body of the manuscript, including formatting related matters.